samples/matter/window_covering/README.rst

```diff
@@ -124,7 +124,11 @@ LED 2:
     Additionally, the LED starts blinking evenly (500 ms on/500 ms off) when the Identify command of the Identify cluster is received on the endpoint ``1``.
     The command's argument can be used to specify the duration of the effect.
 
-LED 3:
+LED 3 (nRF52840 DK and nRF5340 DK):
```

Comment:
```suggestion
LED 3 (nRF52840 DK, nRF5340 DK, and nRF54L15 PDK revision v0.3.0):
```
If we support both revisions.

Reply:
An alternative I just thought of is having this be as it is now in your change, and then we have the second entry be:
```
LED 3 (nRF54L15 PDK revision v0.3.0) or LED 4 (nRF54L15 PDK revision v0.2.1)
```
This way might be more clear that it's the same LED with a different name.